examples/advanced/src/example.tsx

```diff
@@ -1021,8 +1021,7 @@ export default class Example extends React.Component<
               id: 'basemap',
               controller: {
                 type: MapController,
-                // @ts-ignore
-                doubleClickZoom: this.state.mode === 'view' && !this.state.selectionTool,
```

Comment:
I feel like disabling double click to zoom will make nebula feel more snappy and responsive. In my opinion, the benefits of this change outweigh the utility that double click to zoom provides.